src/transient_groundwater.cpp

```diff
@@ -2,23 +2,54 @@
 #include "add_recharge.hpp"
 #include "update_effective_storativity.hpp"
 
-#include <omp.h>
-
 #include <petscdm.h>
 #include <petscdmda.h>
 #include <petscerror.h>
 #include <petscsnes.h>
 
+#include <omp.h>
+#include <experimental/source_location>
```

Comment:
This is probably a silly idea and I should just make a macro rather than pinning us to C++20.